### ISSUE TITLE
Controller loses its IP address 12 hours after boot

### ISSUE THREAD

**Issue by hansthen** [clustervision/trinity#370]
Controller loses its IP address.

**Comment by themisa** [comment]
This is in r4_fixes, and r5_fixes. r5 doesn't have this problem - different code wrt  network manager

**Comment by hansthen** [comment]
Let's see if this also happens after a reboot. So after a deployment simply reboot the node.

**Comment by themisa** [comment]
today:
[root@master ~]# ssh -Y node002
ssh: connect to host node002 port 22: No route to host
[root@master ~]# cat /trinity/version
* r5_fixes
27d630c

**Comment by themisa** [comment]
After a reboot:
[root@master ~]# rpower node002 reset
node002: reset
[root@master ~]# watch nodestat node002
[root@master ~]# ssh -Y node002
Last login: Thu Jul 30 17:31:24 2015 from 192.168.1.254
[root@controller(amaster) ~]# 

After reboot it works - but what if it loses ip again?

**Comment by themisa** [comment]
check tomorrow (1/8) if it loses IP address

**Comment by hansthen** [comment]
**Version:** r5_fixes 27d630c
**Environment:** amaster
```
[root@controller() ~]# ps -ef | grep dhc
dhcpd     3161     1  0 13:02 ?        00:00:00 /usr/sbin/dhcpd -f -cf /etc/dhcp/dhcpd.conf -user dhcpd -group dhcpd --no-pid
root     14784 14759  0 20:02 pts/0    00:00:00 grep --color=auto dhc
[root@controller() ~]# systemctl restart network
[root@controller() ~]# ps -ef | grep dhc
dhcpd     3161     1  0 13:02 ?        00:00:00 /usr/sbin/dhcpd -f -cf /etc/dhcp/dhcpd.conf -user dhcpd -group dhcpd --no-pid
root     15247     1  0 20:02 ?        00:00:00 /sbin/dhclient -H controller -1 -q -lf /var/lib/dhclient/dhclient-82dd53a5-88d0-4f76-a988-b7a6336a6cfc-eno1.lease -pf /var/run/dhclient-eno1.pid eno1
root     15417 14759  0 20:03 pts/0    00:00:00 grep --color=auto dhc
```

**Comment by hansthen** [comment]
Fixed in commit ec6bb635 r5_fixes

**Comment by themisa** [comment]
Still has ip more than 12 hrs later.
**Version:** r5_fixes e5f4623

```
[root@controller() ~]# ps -ef | grep dhc
root      4418     1  0 Aug03 ?        00:00:00 /sbin/dhclient -H controller -1 -q -lf /var/lib/dhclient/dhclient-575667c3-97c3-4413-928b-81ac8c8c1b1b-eno1.lease -pf /var/run/dhclient-eno1.pid eno1
root     14542 14317  0 09:59 pts/0    00:00:00 grep --color=auto dhc
dhcpd    17006     1  0 Aug03 ?        00:00:04 /usr/sbin/dhcpd -f -cf /etc/dhcp/dhcpd.conf -user dhcpd -group dhcpd --no-pid
```
